FeatureAreas.props

```diff
@@ -3,12 +3,12 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Feature to include for inner loop build -->
```

Comment:
partial is a bit vague. I think we should name them something along the lines of 
Product, ProductAndTest and None...

Reply:
I like productOnly. Agreed that "partial" isn't very specific.